templates/_layout.html

```diff
@@ -54,6 +54,9 @@
                         <a class="nav-link active" href="/login">Login</a>
                     {%endif%}
                     </li>
+                    <li class="nav-item">
+                        <a class="nav-link active" href="/logout">Logout</a>
+                    </li>
```

Comment:
I don't believe we should add a logout link in the nav bar. 
I think I'd be better as a button on the user profile so that they can only access it once they've signed in. I believe most people wouldn't want to sign out frequently so it can just be tucked away on the user profile page.